docs/content/v1/usage/reactive-style.md

```diff
@@ -1,32 +1,28 @@
 ---
 title: Reactive CSS
-weight: 9
+weight: 10
 ---
 
-CSS is much more fun when it's scoped.
-Here it come with the Web Components.
+CSS is much more fun when it's scoped. Here it comes with the Web Components, thanks to [Shadow DOM encapsulation](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_shadow_DOM)
 
 Here again, no trick, just the full power of Web Components and scoping styles.
-Well, you should know that the css is reactive too! {{< emoji "😲" >}}
 
-Writing CSS is as easy as
+Writing CSS is as easy as:
 
 ```html
 <template>
-  <h1>Bonjour!</h1>
+  <h1>Hello!</h1>
 </template>
 
 <script>
   export default class extends Lego {
-    init() {
-      this.state = { fontScale: 1 }
-    }
+    …
   }
 </script>
 
 <style>
   :host {
-    font-size: ${state.fontScale}rem;

```

Comment:
For the first example on how to set styles in LEGO component, I thought reactive CSS is too advanced. I keep it a bit further in the article.